src/routes/v2/authenticated/sites.ts

```diff
@@ -199,6 +211,8 @@ export class SitesRouter {
       this.authorizationMiddleware.verifySiteMember,
       attachReadRouteHandlerWrapper(this.launchSite)
     )
+    router.post("/preview", attachReadRouteHandlerWrapper(this.getPreviewInfo))
```

Comment:
hey why is this a post call ya? seems more suited for a `get` call!

Reply:
From the convo in the frontend PR:

I didn't find a way to send a list of strings (sites) through a get request, and therefore resorted to a post request. If you know of a way pls lmk

Reply:
made a similar comment in the fe pr, do we really need a list of sites here? 

reason for bringing this up is because GET requests are intended to retrieve data from a server and do not modify the server’s state. On the other hand, POST requests are used to send data to the server for processing and may modify the server’s state!

perhaps can just: 

get request on `/sites/preview`, and return only a list of sites that

has an image that is defined
if admin return empty array so this wouldnt slow down our network calls

our cookies already has the user info, so the FE does not need to compute this

Reply:
answered similarly in frontend pr.

We wish to support Github login users for the time being. Hence, theres a need to send the list of sites from frontend to backend to be requested. In the future, this can be removed and replaced with a db query after all github users are migrated into email users.